panellayout/build/generated/source/buildConfig/debug/com/wayfair/panellayout/BuildConfig.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Automatically generated file. DO NOT MODIFY
```

Comment:
generated file should be ignored. Actually the `build/` folder needs to be ignroed.

Reply:
Hmm I think we need better gitignore